lib/galaxy/config/__init__.py

```diff
@@ -260,6 +260,17 @@ def select_one_or_all_paths_from_list():
     def _path_exists(self, path):  # factored out so we can mock it in tests
         return os.path.exists(path)
 
+    def _set_alt_paths(self, option, *alt_paths):
```

Comment:
Passing `alt_paths` as a list seems more natural to me, but I suppose it's personal preference.

Reply:
I considered it; I also considered checking `if type(alt_paths) is list` inside the function, but finally decided on `*alt_paths`. I think, typically, this function will be called with just one argument - so I put the burden of accommodating flexibility on the function, not the caller. But that's the only reasoning I have - I have no real preference here.